torch/csrc/stable/c/shim.h

```diff
@@ -37,6 +37,34 @@ AOTI_TORCH_EXPORT AOTITorchError torch_library_impl(
     void (*fn)(StableIValue*, uint64_t, uint64_t),
     uint64_t extension_build_version);
 
+struct StableListOpaque;
+using StableListHandle = StableListOpaque*;
+
+// returns an owning reference of a StableList. callee is responsible for
+// freeing memory.
+AOTI_TORCH_EXPORT AOTITorchError
+torch_new_list_reserve_size(size_t size, StableListHandle* ret);
+
+// new list handle steals ownership of underlying elements from orig_handle
+AOTI_TORCH_EXPORT AOTITorchError torch_new_list_handle(
```

Comment:
maybe call it torch_move_list_handle? also this seems like an odd function to need; why can't the caller just continue to use `orig_handle`?

Reply:
yea this was leftover when i had an overcomplicated design for memory, but we do not need it anymore.